### DISCUSSION TITLE
migrate DB from pg 11 to mysql 8.0

### DISCUSSION THREAD

**Discussion by haydenwang** [ory/hydra#3150] [Q&A] [answered]
Hi guys.
Currently we're using PG 11 as the database.
Now we're plannging to change the pg to mysql 8.0.
Are there any issues、risks or suggestiongs for this migration?
Thx a lot.
Regards

**Comment by vinckr** [comment, marked as answer]
Hello @haydenwang,
Not that I am aware of! 
Check out our [Apply Upgrades](https://www.ory.sh/docs/hydra/guides/upgrade) document for general pointers when upgrading/migrating. Please let us know if any issues arise migrating the DB to mysql 8.0!

**Reply by vinckr** [reply]
Hey @haydenwang,
Just checking in if your question was answered or if there was still something unclear.
I will mark this as answered for now, but feel free to come back to the discussion any time.

Thanks a ton! 🐝
